src/OrbitGl/CallstackThreadBar.cpp

```diff
@@ -118,12 +118,15 @@ void CallstackThreadBar::DoUpdatePrimitives(PrimitiveAssembler& primitive_assemb
   const Color kGreyError(160, 160, 160, 255);
   ORBIT_CHECK(capture_data_ != nullptr);
 
+  uint32_t view_element_x = viewport_->WorldToScreen(GetPos())[0];
```

Comment:
This should not be screen pos, but world pos (so `= GetPos()[0]`). You're using this in world space later. It doesn't make a visual difference since there is still a 1:1 mapping between those spaces, but it's conceptually incorrect.

Reply:
https://github.com/google/orbit/pull/4494

Reply:
Thanks Pablo

Reply:
Rebased to use Pablo's fix.